### PR TITLE
Group dashboard

### PR DESCRIPTION
**Pull request by HansolLee1993**
Created group dashboard view.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,8 +7,7 @@
     "require": {
         "php": ">=5.6.4",
         "laravel/framework": "5.4.*",
-        "laravel/tinker": "~1.0",
-        "laravelcollective/html": "^5.4"
```
Comment (nikoootine) [inline composer.json]: You accidentally remove laravelcollective/html.

Comment (HansolLee1993) [inline composer.json]: wow that's why!!! my vagrant is not working now lol

Comment (nikoootine) [inline composer.json]: looool. put it again and then run "composer update"

Comment (nikoootine) [inline composer.json]: can you repush another commit to fix this. I'll approve this PR now

**Comment by nikoootine** [comment]
@HansolLee1993  can you repush the fix that you made for this branch? thanks

**Comment by nikoootine** [comment]
Closing PR. talked offline that this component will be included in Hansol's next push

**Pull request closed without merging**